README.md

```diff
@@ -4,8 +4,20 @@ Persistent cache for facility lookups.
 
 ## Deployment
 
+### Command line
+
 1. Clone this repository.
 1. Run `npm install`.
 1. Create a config file e.g. `production.json` in the `config` directory. See the [default config](https://github.com/jembi/facility-cache/blob/master/config/config.json) for the available options.
 1. Make sure that your `NODE_ENV` environment variable is set correctly e.g. `export NODE_ENV=production`.
 1. Run `npm start`.
+
+### Docker
+
+Use this command to build the Docker image: 
+
+`docker build -t jembi/facility-cache .`
```

Comment:
@Zooloo2014 

Please could you remove the prefix here. It's confusing when listing docker images as images that were actually pulled from jembi's dockerhub repo will be prefixed with `jembi/`


```suggestion
`docker build -t facility-cache .`
```